app/controllers/staff/matched_efforts_controller.rb

```diff
@@ -6,6 +6,13 @@ def new
     @matched_effort = MatchedEffort.new
   end
 
+  def edit
+    @activity = Activity.find(params[:activity_id])
+    authorize @activity
```

Comment:
Likewise this will invoke `ActivityPolicy#edit?` and `ActivityPolicy#update?`, which might be what we need but I wanted to double check.